src/time-picker/demos/enUS/step-time.demo.md

```diff
@@ -0,0 +1,23 @@
+# Step Time
+
+Pass a number as a step or Pass a array as option。
```

Comment:
```suggestion
Pass a number as step or use an array to specify the item you want.
```

@07akioni You can use the suggestion feature of GitHub, in case you don't know.

Reply:
I knew it

Reply:
> I knew it

I'd like to hear the reason why you don't use it :thinking: